lib/components/app/call-taker-panel.js

```diff
@@ -147,12 +147,11 @@ class CallTakerPanel extends Component {
     const { expandAdvanced, planTripClicked } = this.state
     const advancedSearchStyle = {
       background: 'white',
+      boxShadow: 'none',
       display: expandAdvanced ? undefined : 'none',
```

Comment:
Actually, `expandAdvanced` into whether to render the thing at all.

Reply:
I think I tried this and it causes problems with the state because the forms disappear